test/frontend_test.dart

```diff
@@ -0,0 +1,457 @@
+//@Skip()
+@Timeout(const Duration(seconds: 45))
+import 'angular_utils.dart';
+import 'testing_utils.dart';
+import 'dart:io';
+import 'package:webdriver/io.dart';
+import 'package:test/test.dart';
+
+void main() {
+  WebDriver driver;
+  const String url = "https://www.gyerunkanyukam.hu/";
+  var newUserEmail = "connor.stansfield.development@gmail.com";
+  var oldUserEmail = "connor_stansfield@live.com";
+  var allUserPassword = "password123%";
+
+  group("http client |", () {
+    group("request response tests |", () {
+      var responseCode;
+      HttpClient client;
+
+      setUp(() {
+        client = new HttpClient();
+      });
+
+      tearDown(() {
+        client.close();
+      });
+
+      test('loading site returns valid content', () async {
+        await client.getUrl(Uri.parse(url)).then((HttpClientRequest request) {
+          return request.close();
+        }).then((HttpClientResponse response) {
+          responseCode = response.statusCode;
+        });
+        expect(responseCode == 200, true);
+      });
+    });
+  });
+
+  group("web driver |", () {
+    setUpAll(() async {
+      startSelenium();
+      driver = await setupWebDriver(1280, 1024);
+    });
+
+    tearDownAll(() {
+      stopSelenium();
+      driver.quit();
+      playCompletionBeep();
+    });
+
+    group("basic navigation tests |", () {
+      setUp(() async {
+        await driver.get(url);
+        await waitForAngular(driver);
+      });
+
+      test('base url navigates home', () async {
+        expect(await driver.currentUrl, endsWith('/home'));
+      }, timeout: new Timeout.factor(2));
+
+      test('there is no error messages in the console', () async {
+        bool hasErrors = false;
+        await sleep5();
+        hasErrors = await checkForConsoleErrors(driver, 'SEVERE');
+        expect(await hasErrors, false);
+      });
+
+      test('at least 4 elements in the szolgaltatasok menu', () async {
+        var servicesBtn = await driver.findElement(
+            const By.cssSelector('div.menu-item.menu-item--services > a'));
+        await servicesBtn.click();
+        await sleep2();
+        var menuItemsLength = await driver
+            .findElements(const By.cssSelector(
+                'div.menu-item.menu-item--services > div.nav-menu > div.menu-item'))
+            .length;
+        print("Services menu elements found: " + menuItemsLength.toString());
+        expect(await isXAtLeastValueOfY(menuItemsLength, 4), true);
```

Comment:
No need to `await`, and you can use the build in `greaterThanOrEqualTo` matcher:

```
expect(menuItemsLength, greaterThanOrEqualTo(4));
```